jobs/shared/build.yml

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) Microsoft Corporation.
+
+jobs:
```

Comment:
If you want to break these all into separate job/task files I don't mind, but it's probably not really necessary for such a small build.

Reply:
I felt like I might as well, because I needed to share the jobs.